examples/react-app/src/App.tsx

```diff
@@ -10,27 +10,32 @@ import {
   WalletMultiButton,
 } from "@solana/wallet-adapter-react-ui";
 import { Example } from "./Example";
+import { QueryClientProvider, QueryClient } from "@tanstack/react-query";
 
 // Default styles that can be overridden by your app
 import "@solana/wallet-adapter-react-ui/styles.css";
 
 export const RPC_URL = import.meta.env.VITE_RPC as string;
 
+const queryClient = new QueryClient();
```

Comment:
WDYT about exposing client and query keys to the world, so users can invalidate them manually? For example someone (at least us) is using 'useRecordsV2', and also has functionality to update them. Once records updated, it will be good to be able to invalidate 'useRecordsV2', because right now as I can see they will be invalidated only on the component's re-mount